packages/app-player/tests/packets.spec.ts

```diff
@@ -1,11 +1,9 @@
 import { play } from 'app-player';
 import { getPath } from '@brimdata/sample-data';
 
-// Timeouts are increased due to observed long pcap load times in CI.
-// See https://github.com/brimdata/zui/pull/2978
```

Comment:
This comment is related to timeout increases that were indeed added back in #2978 but the timeouts were dropped when the test was recently simplified, so the comment should go too.